src/utils.js

```diff
@@ -36,42 +36,47 @@ import {
 
 type Section = 'pull_request' | 'push';
 type GeraldFile = 'NOTIFIED' | 'REVIEWERS';
-type NameAndLabelToFiles = {[nameAndLabel: string]: string[], ...};
+type NameToLabelToFiles = {[name: string]: {[label: string]: string[], ...}, ...};
 type CommentHeaders = 'Reviewers' | 'Required Reviewers' | 'Notified';
 
 /**
  * @desc Make the comment body for each of the Gerald sections.
  *
- * @param peopleToFiles - List of people being notified / requested for review and
+ * @param peopleToLabelToFiles - List of people being notified / requested for review and
  * the files they are being notified for / reviewing.
  * @param sectionHeader - What part of the Gerald comment are we making a section for?
  */
 export const makeCommentBody = ({
-    peopleToFiles,
+    peopleToLabelToFiles,
     header,
     tagPerson,
 }: {
-    peopleToFiles: {[string]: Array<string>, ...},
+    peopleToLabelToFiles: NameToLabelToFiles,
     header: CommentHeaders,
     tagPerson: boolean,
 }) => {
-    const names: string[] = Object.keys(peopleToFiles);
-    if (names.length) {
-        let body = `<details>\n<summary><b>${header}</b></summary>\n\n`;
-        names.forEach((person: string) => {
-            const files = peopleToFiles[person];
+    const names: string[] = Object.keys(peopleToLabelToFiles);
```

Comment:
I bet this impacts the #remove-me functionality. Have you checked that? This is big enough that it's fine to do that part in another PR if it's easier.

Reply:
I did check that, at least by reading the code.  My changes do not affect `#remove-me` as far as I can tell.  The remove-me logic goes through getFilteredLists(), and that works on the key to peopleToLabelToFiles, which is still the person's name.  (My previous way of doing this _did_ break remove-me, but this way should not.)

Why did you think this would impact remove-me?

Reply:
> Why did you think this would impact remove-me?
I guess I thought it might impact the remove-me function because I didn't really know how to remove me function works… I just had a hunch that it might, and I wanted to make sure that it was considered and tested.

Sounds like the way that the functionality works is that it just re-creates the comment minus the removed person? So if the removed person was originally in there twice, and they wanted to be removed, it just leave them out for both rules? Instead of, say, matching the name in the body of the previous comment.

Reply:
That's right.

Reply:
#removeme